pyta.py

```diff
@@ -36,7 +48,13 @@ def check(module_name):
                                 'checkers/IO_Function_checker'])
     linter.read_config_file()
     linter.load_config_file()
-    linter.check([spec.origin])
+
+    # When module is not found, raise exception.
+    try:
+        linter.check([spec.origin])
+    except AttributeError:
+        print ("The Module '{}' could not be found in the given "
```

Comment:
you can bind this error to a variable,

and use `sys.exc_info()` from the `sys` package to get the object that caused the error, and the line number stored in the `tb_lineno` property of the traceback object (the third element ing the tuple returned by `sys.exc_info()`)